tests/conftest.py

```diff
@@ -21,6 +21,16 @@
 
 BINDIR = os.path.join(os.path.abspath(os.environ['PWD']))
 
+# List of environment variables copied during hlwm process creation:
+# * LSAN_OPTIONS: needed to suppress warnings about known memory leaks
+COPY_ENV_WHITELIST = ['LSAN_OPTIONS']
+
+
+def extend_env_with_whitelist(environment):
```

Comment:
Should I extract this into a separate PR?

Reply:
It's fine as it is! :)

Reply:
Alright, thanks! :)